source/_addonStore/dataManager.py

```diff
@@ -142,10 +157,10 @@ def _getCachedAddonData(self, cacheFilePath: str) -> Optional[CachedAddonsModel]
 			cacheData = json.load(cacheFile)
 		if not cacheData:
 			return None
-		fetchTime = datetime.fromisoformat(cacheData["cacheDate"])
+		cacheHash = cacheData.get("cacheHash")
 		return CachedAddonsModel(

```

Comment:
Now that we are changing the cache structure on alpha, we should make sure that if the stored cache doesn't fit the current cache schema, we return none so that NVDA refreshes the cache.

This return statement could be wrapped in a try/except, alternatively, we can create a [JSON Schema](https://json-schema.org/) for validation.